examples/node/utils.ts

```diff
@@ -1,12 +1,26 @@
-import { type CasinoChain } from "@betswirl/sdk-core";
+import {
+  type CasinoChain,
+  type CasinoChainId,
+  casinoChainById,
+  casinoChains,
+} from "@betswirl/sdk-core";
 import { createConfig, webSocket } from "@wagmi/core";
 import { http, type Hex, createPublicClient, createWalletClient } from "viem";
 import { privateKeyToAccount } from "viem/accounts";
 
+export function getWagmiConfigForAllChains() {
+  return createConfig({
+    chains: [casinoChains[0].viemChain, ...casinoChains.slice(1).map((c) => c.viemChain)],
```

Comment:
Is it the same as
```suggestion
    chains: casinoChains.map((c) => c.viemChain),
```
?

Reply:
![image](https://github.com/user-attachments/assets/77edb2db-e4de-4367-a48c-7718ac23af01)

This is what I tried to do, but it does not work, th syntax of `createConfig` is very special, idk why